evaluation/poets/scripts/1syllable_words.sh

```diff
@@ -0,0 +1,3 @@
+# 1-syllable words 
+INPUT=${INPUT:-inputs/genesis}
```

Comment:
Similarly, I would replace `inputs/genesis` with the path relative to `$PASH_TOP`.

Reply:
(Update for @angelhof : we are moving inputs to point to `$PASH_TOP/evaluation/scripts/input/` and updating `gen.sh` to pull the right inputs for these scripts.)